spec/unit/installer_spec.rb

```diff
@@ -86,21 +86,21 @@ def @installer.clean_pod_sources
       end
 
       it 'in runs the post-install hooks before serializing the Pods project' do
-        @installer.stubs(:prepare_pods_project)
         @installer.stubs(:run_podfile_pre_install_hooks)
-        @installer.stubs(:install_file_references)
-        @installer.stubs(:install_libraries)
-        @installer.stubs(:set_target_dependencies)
         @installer.stubs(:write_lockfiles)
         @installer.stubs(:aggregate_targets).returns([])
         @installer.unstub(:generate_pods_project)
+
+        Pod::Xcode::PodsProjectGenerator.any_instance.stubs(:generate!)
+        Pod::Xcode::PodsProjectGenerator.any_instance.stubs(:share_development_pod_schemes)
+        Pod::Xcode::PodsProjectGenerator.any_instance.stubs(:write) do
```

Comment:
@segiddins I don't think this is the right way to stub this, do you know of a better way?